LayoutTests/imported/w3c/web-platform-tests/css/css-masking/parsing/mask-valid.sub.html

```diff
@@ -11,52 +11,50 @@
 </head>
 <body>
 <script>
-// <mask-layer> = <mask-reference> <masking-mode>? || <position> [ / <bg-size> ]? ||
-// <repeat-style> || <geometry-box> || [ <geometry-box> | no-clip ] || <compositing-operator>
+// <mask-layer> = <mask-reference> || <position> [ / <bg-size> ]? || <repeat-style> ||
+// <geometry-box> || [ <geometry-box> | no-clip ] || <compositing-operator> || <masking-mode>
 
-// <mask-reference> <masking-mode>?
 // <mask-reference> = none | <image> | <mask-source>
-// <masking-mode> = alpha | luminance | auto
 test_valid_value('mask', 'none');
-test_valid_value('mask', 'none alpha');
```

Comment:
I didn’t exactly **remove** it. I simply changed the test to just specify `alpha` rather than `none alpha` and moved it down below to the `<masking-mode>` section. When the test was originally written, the `<mask-reference>` was mandatory in the specification and couldn’t be omitted. There would be no problem including a `none alpha` as well as an `alpha` test, but there really weren’t too many of those here, with initial values intended to be omitted, so I didn’t think it should be added. Happy to add it if you like.